README.md

```diff
@@ -42,6 +42,9 @@ Notes:
   Amber indicates that some verification has been performed, but the configuration is still experimental.
   Red indicates a configuration with minimal/no verification.
   Users must make their own assessment of verification readiness for any tapeout.
+* Bit-Manipulation support implements v0.92 of the specification.
```

Comment:
The phrasing seems slightly awkward here. Maybe put it the other way round - e.g. "v0.92 of the RISC-V Bit Manipulation Extension is implemented."